docs/testing.rst

```diff
@@ -137,6 +147,31 @@ submitting a pull request, or be prepared to be mail-spammed by CI.
 Please note that in practice you're only supposed to submit pull requests
 against the ``develop`` branch, see :ref:`contributing`.
 
+Building documentation
+----------------------
+Eve uses Sphinx_ for its documentation. To build the documentation locally,
+switch to the ``docs`` folder and run ::
+
+    $ make html
+
+This will generate html documentation in the folder ``~/code/eve.docs/html``,
+which can be overridden with the ``BUILDDIR`` make variable ::
+
+    $ make html BUILDDIR=/path/to/my/docs
+
+Make sure Sphinx_ reports no errors or warnings when running the above.
+
+To preview the documentation, switch to the build directory and run ::
+
+    $ python3 -m http.server
```

Comment:
I usually simply do `$ open <path to index.html>`, which does not need an http server. Might be a good idea to add this option too.